src/components/splash/Splash.js

```diff
@@ -1,50 +1,67 @@
-import React, { Component } from 'react'
-import { Image, StyleSheet, View } from 'react-native'
+import React from 'react'
+import { Image, StyleSheet } from 'react-native'
 import normalize from 'react-native-elements/src/helpers/normalizeText'
-import splashImage from '../../assets/splash.png'
-import { Text } from '../common'
+import splashImage from '../../assets/Splash/logo.svg'
+import goodDollarImage from '../../assets/Splash/goodDollar.svg'
+import wavePattern from '../../assets/wave.svg'
+import Wrapper from '../common/layout/Wrapper'
+import Section from '../common/layout/Section'
 
 //minimize delay <Image> has over web <img>
 Image.prefetch(splashImage)
-class Splash extends Component {
-  render() {
-    return (
-      <View style={styles.screen}>
-        <Text style={styles.splashText}>
-          {`Welcome and thank you for participating in GoodDollar's \n`}
-          <Text style={styles.boldText}>Early Access Alpha</Text>
-        </Text>
+Image.prefetch(goodDollarImage)
+Image.prefetch(wavePattern)
+
+const Splash = () => (
+  <Wrapper style={styles.container}>
+    <Wrapper style={styles.wrapper}>
```

Comment:
`Wrapper` inside `Wrapper` seems weird. Couldn't we remove one?

Reply:
:point_up_2: